src/shader_recompiler/frontend/translate/vector_alu.cpp

```diff
@@ -595,4 +604,27 @@ void Translator::V_CMP_CLASS_F32(const GcnInst& inst) {
     }
 }
 
+void Translator::V_MAD_U32(const GcnInst& inst) {
+    const IR::U32 src0{GetSrc(inst.src[0])};
+    const IR::U32 src1{GetSrc(inst.src[1])};
+    const IR::U32 src2{GetSrc(inst.src[2])};
+    // TODO
+    const IR::U32 result = ir.IAdd(ir.IMul(src0, src1), src2);
+    SetDst(inst.dst[0], result);
+    ir.SetVccLo(result);
+}
+
+void Translator::V_MBCNT_U32_B32(bool is_low, const GcnInst& inst) {
+    const IR::U32 src0{GetSrc(inst.src[0])};
+    const IR::U32 src1{GetSrc(inst.src[1])};
```

Comment:
This implementation was tailored for NVIDIA (warp size 32). Query the subgroup size from vulkan backend, pass it in the profile structure and assert for it here